example/Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5.1'
+gem 'rails', '~> 4.2'
 gem 'sqlite3'
 
 gem 'dotenv-rails'
-gem 'shopify_app', '~> 7.0.0'
+gem 'shopify_app', '~> 7.2.0'
```

Comment:
> because of SQLite, this can't be run on Heroku.

oh right... never looked at the example's `database.yml` before.. always assumed it to be PostgreSQL as well.. :+1: